docs/eventlisteners.md

```diff
@@ -620,7 +620,7 @@ The following pipeline metrics are available on the `eventlistener` Service on p
 |  Name | Type | Labels/Tags | Status |
 | ---------- | ----------- | :-: | ----------- |
 | `eventlistener_triggered_resources` | Counter | `kind`=&lt;kind&gt; | experimental |
-| `eventlistener_event_count` | Counter | `status`=&lt;status&gt; | experimental |
+| `eventlistener_event_received_count` | Counter | `status`=&lt;status&gt; | experimental |
```

Comment:
Hi @sudhishmk :wave: 
I m not able to understand here how the metrics issue getting solved by changing the name

Reply:
@savitaashture  Issue seems to be caused by conflict with this [metric](https://github.com/tektoncd/triggers/blob/9b472899801f88cd700fe3b99bf6bb7fc88af494/vendor/knative.dev/eventing/pkg/metrics/source/stats_reporter.go#L32-L36) with same name.

Reply:
`knative/pkg` has added a new metric with the same name. So we need to change our metrics to some other name.

Reply:
ah ok got it :+1: